API/API.csproj

```diff
@@ -16,6 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Models\Models.csproj" />
```

Comment:
I'm not sure whether we really need a separate project for models, I would say more no than yes, but left it for simplicity